repo-info/README.md

```diff
@@ -0,0 +1,21 @@
+# Repo Info Helper
+
+This script scans a given .csv file (works on both `pr-data.csv` and `py-data.csv`), specifically, just a single column inside, and outputs another .csv file, with 3 columns:
+
+* Repo URL
+* Months since latest commit to master/main
+* Number of stars
+
+... which are sorted in descending order of number of stars and ascending order of months since last commit to master.
+
+The latter 2 values will help us in shortlisting a project to fix flaky tests in. The chances of your PR getting accepted are higher for a repository that is actively maintained and has a high number of stars.
+
+## To run:
+
+* Required libraries to install: `pandas`, `tqdm`, `pygithub`
```

Comment:
Please add `requirements.txt` as common for Python projects.

Reply:
Should you also remove this text?

Reply:
Yes, thank you. Have removed it and committed.